FirebaseInstanceID.podspec

```diff
@@ -19,6 +19,7 @@ services.
   }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.14'
```

Comment:
The rest of Firebase has `s.osx.deployment_target = '10.11'`.  Will 10.11 work for InstanceID and Messaging?

Reply:
the Apple's API to request notification in mac is 10.14+
https://developer.apple.com/documentation/appkit/nsapplication/2967172-registerforremotenotifications?language=objc

messaging probably won't do anything below 10.14, for InstanceID, it can work in 10.11, in case other SDKs like IAM want to use it for lower versions.

Reply:
In that case, we'll have to disable the Messaging tests on travis - probably by removing them from the AllUnitTests_macos scheme since travis is running 10.13 until we're able to migrate it to Xcode 10.2.   See #2746.

Reply:
I wonder, can the [older API](https://developer.apple.com/documentation/appkit/nsapplication/1428476-registerforremotenotificationtyp?language=objc) be used for previous macOS versions?

Reply:
Yes I think so. This PR only makes fcm and iid compile, not functional, so I guess it doesn't hurt to change both to 10.11. Would it be easy to adjust the support number later?

Reply:
We shouldn't merge to master until it is functional, since master is intended to be always shippable. It should stay on a branch until then.

Reply:
@paulb777 @maksymmalyhin I just test sending a push notification using FCM and succeeds, but there's still some code change, but I can confirm we don't need 10.14 because the older API works.

Reply:
It is functional now, @paulb777 when you have a chance, we should adjust the IID osx version back to 10.11 in stagingSpec, but we can do this after you are done with the release.

Reply:
Done. https://github.com/firebase/SpecsStaging/commit/59cb1e98ea96ae1d2d9c03cc9421866fa7c918a1